docs/sw360-integration.md

```diff
@@ -0,0 +1,66 @@
+# SW360 integration
+
+This guide shows the possible ways to integrate SW360 in the ORT run. The order of the sections is the recommended 
+order in how to use the several ways.
+
+Since the [Eclipse project antenna](https://github.com/eclipse/antenna) is archived, it is also recommended using the 
+SW360 integration with ORT.
+
+## UploadResultToSw360Command
+
+The [`UploadResultToSw360Command`](../cli/src/main/kotlin/commands/UploadResultToSw360Command.kt) is a standalone ORT
+command, which takes an ORT result file as an input and creates components/releases in SW360 for the packages, 
+afterwards it creates the projects and links the created releases to the respective project.
+
+### Command Line
+
+In order to establish a connection to the desired SW360 instance, the 
+[`sw360Configuration`](../model/src/test/assets/reference.conf) has to be specified as a scanner storage in the 
+`ort.conf` file. To use the `upload-result-to-sw360` command with the `ort.conf` file put it to `${HOME}/.ort/conf` or 
+pass it to the ORT command with the `--config` option.
+
+```bash
+cli/build/install/ort/bin/ort upload-result-to-sw360
+  -i [analyzer-output-dir]/analyzer-result.yml
+```
+
+## SW360 curations
+
+### When to use
+
+This can be used in order to correct package metadata using the SW360 web frontend. Currently, it is only possible to 
+do curations for the fields`declaredLicenses`, `homepageUrl`, `binaryArtifact` and `sourceArtifact`. For all the other 
+fields in SW360 there are unfortunately no matching fields in ORT, so they are ignored. The release in SW360 needs an 
+approved clearing state, otherwise the curated data will not be taken.
+
+### Command Line
+
+To enable the usage of the SW360 curations, add the option `--sw360-curations` to the _analyzer_ command. In addition, 
+the [`sw360Configuration`](../model/src/test/assets/reference.conf) has to be specified for the analyzer. 
+
+```bash
+cli/build/install/ort/bin/ort analyze
+  -i [source-code-of-project-dir]
+  -o [analyzer-output-dir]
+  --sw360-curations
+```
+
+## SW360 storage backend
+
+It is possible to use SW360 as a [`ScanResultStorage`](../scanner/src/main/kotlin/ScanResultsStorage.kt), i.e. during 
+the _scanner_ command generated scan results of each package will be uploaded to SW360. Besides uploading, the scan 
+results can also be read in future scans to speed it up. The [SW360 attachment type][1] of the uploaded scan results 
+is `SCAN_RESULT_REPORT`.
```

Comment:
it should be "component license information" or "componen license information (xml)" for pressing as SPDX file